mmv1/third_party/terraform/tests/resource_cloud_run_service_secret_volume_test.go.erb

```diff
@@ -0,0 +1,122 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' %>
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccCloudRunService_secretVolume(t *testing.T) {
```

Comment:
This test can be added to the file containing the other test. Actually both these can be added to the existing cloud run service test that is handwritten.

Reply:
Done.